### PR TITLE
hw

### PR DESCRIPTION
**Pull request by denissmaksimkins**

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	"testing"
+)
+
+func TestIsValidNumber(t *testing.T) {
+	for _, tc := range []struct {
+		num  string
+		base int
+		want bool
+	}{
+		{"", 0, false},
```
Comment (zayac) [inline main_test.go:13]: Can you add a test for an empty string with a valid base (i.e. base > 2). The result should be false.
